DEFAULT_CONFIG.json5

```diff
@@ -74,115 +74,115 @@
       /// The routing strategy to use in peers and it's configuration.
       peer: {
           /// The routing strategy to use in peers. ("peer_to_peer" or "linkstate").
-          mode: "linkstate",
+          mode: "peer_to_peer",
       },
   },
 
   /// The declarations aggregation strategy.
   aggregation: {
       /// A list of key-expressions for which all included subscribers will be aggregated into.
       subscribers: [
-        // key/expression
+        // key_expression
       ],
       /// A list of key-expressions for which all included publishers will be aggregated into.
       publishers: [
-        // key/expression
+        // key_expression
       ],
   },
```

Comment:
Shouldn't this also commented out by default?

Reply:
since the default is an empty array, which is the case here, I'm ok with it not being commented, but having some examples KEs to give a vibe. They do need to be quoted though.